llvm/lib/Target/X86/X86.h

```diff
@@ -104,7 +105,19 @@ FunctionPass *createX86LowerTileCopyPass();
 /// CALL instruction. The pass does the same for each funclet as well. This
 /// ensures that the open interval of function start and end PCs contains all
 /// return addresses for the benefit of the Windows x64 unwinder.
-FunctionPass *createX86AvoidTrailingCallPass();
+class X86AvoidTrailingCallPass
+    : public PassInfoMixin<X86AvoidTrailingCallPass> {
+private:
+  const X86TargetMachine *TM;
```

Comment:
Premerge CI is failing because `TM` is unused. We should be able to remove it from here, the constructor, and drop the `this` argument to the constructor call for this pass in `X86PassRegistry.def`.